base/src/org/compiere/model/Tax.java

```diff
@@ -538,7 +540,7 @@ protected static int get (Properties ctx,
 			//
 			if (tax.getC_TaxCategory_ID() != C_TaxCategory_ID
 				|| !tax.isActive() 
-				|| tax.getParent_Tax_ID() != 0)	//	user parent tax
+				|| !tax.isSummary())	//	#2434 - only looking for summary taxes 

```

Comment:
- The old code continue when a tax have a parent tax
- The new code continue when a tax is not a summary

I think that some is wrong here

Reply:
Should be tax.isSummary() instead !tax.isSummary()